### PR TITLE
add pound as mass

### PR DESCRIPTION
**Pull request by hellow554**

### CODE REVIEW

File: src/mass/index.ts [hunk 1]
```diff
@@ -18,6 +18,8 @@ export function withValueType<NumberType>(arithmetic: Arithmetic<NumberType>) {
     static kilograms = makeUnit('kg', dimension.Mass);
 
     static grams = WithValueType.kilograms.times(1e-3).withSymbol('g');
+
+    static pounds = WithValueType.kilogram(4.535924e-1).withSymbol('lb');
```
Comment (buge) [inline src/mass/index.ts:22]: Can you please use 0.45359237 which is the official definition of the pound?
https://en.wikipedia.org/wiki/International_yard_and_pound

**Comment by buge** [comment]
Thanks a lot for the pull request and super sorry for missing it before!

File: src/mass/index.ts (hunk 1 above)
Comment (hellow554) [reply src/mass/index.ts]: Done. I don't think that it matters (it just rounds it) but thanks for your review :)

File: src/mass/index.ts [hunk 2]
```diff
@@ -18,6 +18,8 @@ export function withValueType<NumberType>(arithmetic: Arithmetic<NumberType>) {
     static kilograms = makeUnit('kg', dimension.Mass);
 
     static grams = WithValueType.kilograms.times(1e-3).withSymbol('g');
+
+    static pounds = WithValueType.kilograms(4.5359237e-1).withSymbol('lb');
```
Comment (buge) [inline src/mass/index.ts:22]: Should be `WithValueType.kilograms.times(4.5359237e-1).withSymbol('lb');`

Comment (hellow554) [reply src/mass/index.ts]: I'm not sure what I did here, but I'm somewhat sorry for all the noise. Let's hope it works this time.

Comment (buge) [reply src/mass/index.ts]: No worries at all. Thanks a lot for the contribution!

**Pull request merged**